Updated invisorql to the 2.0 release.

morganestes opened the pull request:
This also adds known dependencies on OS version, architecture, and an additional
package that the quicklook requires to operate.
The `ffmpeg` package is used if installed, but doesn't affect the plugin, so it's
not on the dependencies list.

morganestes commented:
Hm, I squashed my commits but wasn't given the open to squash the merge of master. Sorry about that, I know squashing is important to PRs.

vitorgalvao commented on Casks/invisorql.rb:
```diff
@@ -1,11 +1,16 @@
+  depends_on :macos => '>= :lion'
+  depends_on :arch => :x86_64
+  depends_on :cask => 'invisor-lite'
+  caveats 'If ffmpeg is installed, invisorql will use it to create thumbnails in QuickLook.
```
Please remove the caveat. These shouldn’t be used for suggestions.

vitorgalvao commented:
Are you having trouble squashing?

morganestes commented:
I removed the caveats line and updated the branch/PR.

I followed the squash guide linked to in the contributing doc and was able to squash my own commits, but was surprised to see the merge to/from master dated Nov. 10, 2014, especially since I opened this branch today. Rebasing didn't show me those commits in the list to pick, or I would've squashed them, too. Any advice on how to keep those merges from showing up in the PRs?

vitorgalvao commented:
It’s an obtuse process. Doing it a few more times will make it clearer. Merging in the above mentioned pull request. Your contribution is still included (and still credited to you), with the appropriate modifications. Thank you.
